governance/votes/Executive vote - August 18, 2023.md

```diff
@@ -0,0 +1,225 @@
+|VoteWizard|[0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) | 9.92 |
+| Libertas | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | 9.92 |
+|Harmony|[0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2](https://etherscan.io/address/0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2) | 8.93|
+| JAG | [0x58d1ec57e4294e4fe650d1cb12b96ae34349556f](https://etherscan.io/address/0x58d1ec57e4294e4fe650d1cb12b96ae34349556f) | 7.61 |
+| Cloaky | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) | 4.30 |
+| Skynet | [0xd4d1A446cD5976a11bd32D3e815A9F85FED2F9F3](https://etherscan.io/address/0xd4d1A446cD5976a11bd32D3e815A9F85FED2F9F3) | 3.64 |
+
+### Direct Deposit Module (D3M) Parameter Housekeeping
+
+As per this [forum post](https://forum.makerdao.com/t/notice-of-executive-vote-date-change-and-housekeeping-changes/21613), the following housekeeping changes to the now defunct D3Ms will be made if this executive proposal passes.
+
+#### Aave D3M
+
+- Remove DIRECT-AAVEV2-DAI from the [DC-IAM (autoline)](https://manual.makerdao.com/module-index/module-dciam).
+- Set the DIRECT-AAVEV2-DAI [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to **0 DAI**.
+
+#### Compound D3M
+
+- Remove DIRECT-COMPV2-DAI from the [DC-IAM (autoline)](https://manual.makerdao.com/module-index/module-dciam).
+- Set the DIRECT-COMPV2-DAI [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to **0 DAI**.
+
+For completeness, the [Global Debt Ceiling (`Line`)](https://manual.makerdao.com/parameter-index/core/param-global-debt-ceiling) will be reduced by as part of this process to account for the removal of the Debt Ceiling of these collaterals.
+
+### New Silver (RWA002-A) Parameter Changes
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmaU1eaD) and the execution of the updated `doc` parameter in the preceding spell, the following parameter changes to the New Silver (RWA002-A) vault will be made, if this executive proposal passes.
+
+- Increase RWA002-A Debt Ceiling by 30 million DAI from 20 million DAI to **50 million DAI**.
+- Increase RWA002-A Stability Fee by 3.5% from 3.5% to **7%**.
+- Reduce Liquidation Ratio by 5% from 105% to **100%**.
+- Bump the Oracle Price to **92,899,356 DAI** (rounded value, see technical note below).
+
+*Technical Note:* the increase in the Oracle Price is required to account for borrowing and accumulated Stability Fees throughout the loan term. Per the Legal Agreement, the loan term runs until 2032-10-11. At the smart contract level, the Oracle Price will be internally set to `92899355926924134500000000`.
+
+Please see the [proposal thread](https://forum.makerdao.com/t/rwa-002-new-silver-restructuring-risk-and-legal-assessment/21417) before voting.
```

Comment:
https://forum.makerdao.com/t/notice-of-executive-vote-date-change-and-housekeeping-changes/21613#rwa002-a-liquidation-ratio-update-3 containts the liquidation ratio change - add link?

Reply:
Ah, here it is. Disregard previous comment

Reply:
added a note